lib/jxl/modular/encoding/enc_encoding.cc

```diff
@@ -248,7 +250,7 @@ Status EncodeModularChannelMAANS(const Image &image, pixel_type chan,
         pixel_type_w topleft = (x && y ? *(r + x - 1 - onerow) : left);
         int32_t guess = ClampedGradient(top, left, topleft);
         int32_t residual = r[x] - guess;
-        tokens->emplace_back(tree[0].childID, PackSigned(residual));
+        (*tokens)[pos++].value = PackSigned(residual);
```

Comment:
Couldn't we just reserve() instead of Resize()-ing, and leave emplace_back?

Reply:
We already did reserve(), but this is faster (probably because there's no size check).

Reply:
How much faster are we talking about?

Reply:
I remember having a noticeable difference, I don't remember how much. I will try it when I get the chance.

Reply:
Speed goes down to about 33 MP/s on that image (from about 35 MP/s), if I keep the emplace_back.

Reply:
Fair enough. Might be worth leaving a comment that this is noticeably faster in some cases.